### ISSUE TITLE
training  parameter problems

### ISSUE THREAD

**Issue by TinkerBell123** [facebook/C3D#49]
hello Du，When i train the 8 conv networks，I met a problem.Subject to my computing,i set the train and test proto batch size to 5,the lr is 0.001,the max_iter is 100000,the stepsize is 20000.when the network run to 40000 iterations,the loss still around 4.6,not converge,not diverge.could give me some advises about the parameters?Best Wishes,thank you very much!

**Comment by dutran** [comment]
@TinkerBell123 what do you mean by "the 8 conv networks"? and which dataset you are training on?

**Comment by TinkerBell123** [comment]
I use ucf101 dataset.
when I first train your network,the proto file in c3d_train_ucf101/conv3d_ucf101_train.prototxt has 5 convolutional network layers,I later found the proto files have 8 convolutional layers in finetuning and feature extraction,so I change the 8 conv layers prototxt file to train ucf101.The first I used the small networks,it converge well.But when I change the larger one ,it ocurrs the problem

**Comment by TinkerBell123** [comment]
I know what's the problem,thank you very much

**Comment by dutran** [comment]
Glad to hear.

**Comment by haohao900618** [comment]
hello @TinkerBell123 I have met the same problem like you. I met the problem when I run c3d_train_ucf101/train_ucf101.sh https://github.com/facebook/C3D/issues/85. I want to know how did you solve the peoblem. Thank you very much!
